tests/lib/signIn.js

```diff
@@ -158,35 +160,40 @@ define([
 
         return accountHelper.newVerifiedAccount()
           .then(function (account) {
+            var numSetupRequests = requests.length;
             var incorrectCaseEmail = account.input.email.charAt(0).toUpperCase() + account.input.email.slice(1);
 
-            return respond(client.signIn(incorrectCaseEmail, account.input.password), RequestMocks.signIn);
```

Comment:
As far as I can tell, this test wasn't actually exercising the "incorrect case" code-path.